plugins/catalog-backend-module-ldap/src/processors/LdapOrgEntityProvider.ts

```diff
@@ -36,6 +38,54 @@ import {
   UserTransformer,
 } from '../ldap';
 
+/**
+ * Options for {@link LdapOrgEntityProvider}.
+ *
+ * @public
+ */
+export interface LdapOrgEntityProviderOptions {
+  /**
+   * A unique, stable identifier for this provider.
+   *
+   * @example "production"
+   */
+  id: string;
+
+  /**
+   * The target that this provider should consume.
+   *
+   * Should exactly match the "target" field of one of the "ldap.providers"
+   * configuration entries.
+   *
+   * @example "ldaps://ds-read.example.net"
+   */
+  target: string;
+
+  /**
+   * The logger to use.
+   */
+  logger: Logger;
+
+  /**
+   * The refresh schedule to use.
+   *
+   * If you pass in 'manual', you are responsible for calling the `read`
+   * method manually at some interval. If not, it will be automatically
+   * called regularly with the given schedule using the scheduler.
+   */
+  schedule: 'manual' | TaskSchedule;
```

Comment:
Perhaps just `schedule?: TaskSchedule`?

Reply:
Well ... i kinda wanted to make it required, to force the user to make a choice. The surprise of adding it and nothing happening, because you forgot to add this field out of many, felt like the worse experience.

Reply:
Yeah that makes sense 👍